Makefile

```
@@ -1,6 +1,6 @@
 source-to-image:
 	find node_modules -name ".git" | rev | cut -c6- | rev | xargs rm -fr
-	npm install
-	npm run build
+	pnpm install --frozen-lockfile
```

Ei tarvi frozen-lockfile-asetusta tässä eikä missään muussakaan `pnpm install`:ssa kun se on .npmrc:ssä.

```suggestion
	pnpm install
```